index.d.ts

```diff
@@ -0,0 +1,65 @@
+import { IncomingMessage, ServerResponse } from 'http';
+import pino from 'pino';
+
+export = PinoHttp;
```

Comment:
Can you do named exports instead? This style of export has issues when esModuleInterop is disabled, AFAIR.